torch/onnx/_internal/fx/context.py

```diff
@@ -0,0 +1,99 @@
+import copy
+from typing import List
+
+import torch
+
+
+class FxToOnnxContext:
```

Comment:
Is this going to be tread safe?

Reply:
It's not right now. I don't see thread-safe is maintained in Meta's `symbolic_trace`, so I assume it's not a problem.